mailbox/backup/src/main/java/org/apache/james/mailbox/backup/Zipper.java

```diff
@@ -99,8 +103,7 @@ private void storeInArchive(MailboxAnnotation annotation, String directory, ZipA
         ZipArchiveEntry archiveEntry = (ZipArchiveEntry) archiveOutputStream.createArchiveEntry(new File(entryId), entryId);
 
         archiveOutputStream.putArchiveEntry(archiveEntry);
-        IOUtils.copy(IOUtils.toInputStream(annotation.getValue().orElse(""), Charsets.UTF_8
-        ), archiveOutputStream);
+        IOUtils.copy(IOUtils.toInputStream(annotation.getValue().orElse(""), Charsets.UTF_8), archiveOutputStream);
```

Comment:
a little nitpicking, but would there be a way to handle in a different way a NIL value (represented here by an empty optional) from an empty annotation?

Reply:
I think that an entry with an empty content, will have the same representation as an entry without content.
I could write a test to validate this assumption.
If it hold true, we should create an issue to handle NIL value, for example with an extra field on the entry.